### ISSUE TITLE
[Feature Request] Preserve heading categories when rolling over

### ISSUE THREAD

**Issue by josephdong1000** [shichongrui/obsidian-rollover-daily-todos#37]
This is kind of a big ask, and I don't really expect this to be implemented anytime soon, but it would be really nice if to-dos could be rolled under the same heading they were under previously. 

Here's one implementation that I thought of:
1. For every incompleted to-do in yesterday's note, get the heading name
2. Check today's note for a matching heading name. If found, dump the to-do under that heading
3. If no heading matches, dump to the bottom of the note

I'd greatly appreciate if something like this was implemented! Thanks!

**Comment by smaroukis** [comment]
I am also looking for this feature!

**Comment by gaoDean** [comment]
Yes same, i just wanted to input that I would very much like that too!

**Comment by dustinmcbride** [comment]
+1

**Comment by tim-hilt** [comment]
Fixed by #80

**Comment by alttext** [comment]
I would also be ok with just limiting the to-do rollover to one heading as well.
